### PR TITLE
TA05 Added Cypress & E2E tests.

### PR DESCRIPTION
**Pull request by devankur03**
- Added Cypress for UI  testing
- Added End to End test

### CODE REVIEW

**Review by 3ankur — COMMENTED** [review]
i have given few comments, please address those.

File: cypress/fixtures/profile.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": 8739,
```
[the remark this answers is not in the document]
Comment (devankur03) [reply cypress/fixtures/profile.json:2]: Yea, it comes by default in Cypress installation

File: cypress/fixtures/users.json [hunk 2]
```diff
@@ -0,0 +1,232 @@
+[
```
[the remark this answers is not in the document]
Comment (devankur03) [reply cypress/fixtures/users.json:1]: Yea, i am not using this anywhere, it comes by default in Cypress install , I'll remove it in the next commit, I am working on some feature tests,  will push in this commit

**Review by 3ankur — APPROVED** [review]
Looks good to me .

**Pull request merged**